src/uwtools/api/driver.py

```diff
@@ -4,6 +4,103 @@
 
 import importlib
 import sys
+from datetime import datetime, timedelta
+from importlib.util import module_from_spec, spec_from_file_location
+from inspect import getfullargspec
+from pathlib import Path
+from typing import Optional, Type, Union
+
+from uwtools.drivers.support import graph
+from uwtools.drivers.support import tasks as _tasks
+from uwtools.logging import log
+from uwtools.utils.api import ensure_data_source
+
+
+def execute(  # pylint: disable=unused-argument
+    module: str,
+    classname: str,
+    task: str,
+    schema_file: str,
+    module_dir: Optional[Union[Path, str]] = None,
```

Comment:
New execute function for `uw invoke`. `cycle` and `leadtime` need to be optional arguments, since we have no way to presume the requirements of the user-made driver. We also needed to add `module`, `classname`, `module_dir` arguments that are absent in other `execute()` functions.

Reply:
I hadn't considered that inconsistency. In that case, I think we should use `uw execute` for this mode.